_includes/master/manifests/calico-kube-controllers.yaml

```diff
@@ -74,6 +74,11 @@ spec:
             # Mount in the etcd TLS secrets.
             - mountPath: /calico-secrets
               name: etcd-certs
+          readinessProbe:
```

Comment:
@caseydavenport what other versions should I apply the changes to or is master enough?

Reply:
master is enough, since the code itself is only in master :)